blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -57,7 +57,23 @@ export const actionHandlers = {
         return state.set('currentRuns', null);
     },
     [ACTION_TYPES.SET_CURRENT_RUN_DATA](state, { payload }): State {
-        return state.set('currentRuns', payload);
+        return state.set('currentRuns', payload.map((run) => {
+            if (run._class === 'io.jenkins.blueocean.service.embedded.rest.QueueItemImpl') {
```

Comment:
And not sure what we can do ... once they are there, people will use them and depend on them.